src/test/java/FatihTest/TestFS1001.java

```diff
@@ -0,0 +1,7 @@
+package FatihTest;
+
+public class TestFS1001 {
+
+    // day 1
+    // some testing has been done
```

Comment:
Hi,
This line is necessary if is not then you can delete and reduce your code?!
Thanks!

Reply:
hi , thanks for the feedback and heads up i will work on that